use/supybot-botchk.rst

```diff
@@ -30,7 +30,7 @@ following content replacing things were suitable::
     Environment="PATH=/usr/local/bin:/usr/local/sbin:/usr/local/games:/usr/bin:/usr/sbin:/usr/games:/bin:/sbin:/bin:/opt/local/bin:/opt/local/sbin:/opt/local/games"
     Environment="TZ=UTC"
     Type=simple
-    ExecStart=/usr/local/bin/supybot /home/bot/botname/botname.conf
+    ExecStart=/usr/bin/supybot /home/bot/botname/botname.conf
```

Comment:
If we're worried about people using the wrong path and getting confused, we could also have something like `/path/to/supybot`, which should make it very obvious that they need to remember how they installed the bot.